How to search and order a phone number guide (#633)
Bandwidth/api-docs: site/docs/numbers/searchingAndOrderingNumbers.mdx

---
id: searchingAdnOrderingNumbers
title: How to search and order a phone number
slug: /numbers/guides/searchingForNumbers
description: How to search for available numbers using the Bandwidth API
keywords:
  - bandwidth
  - numbers
  - search
image: ../../static/img/bandwidth-logo.png
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

export const accountId = "{accountId}";

export const Highlight = ({children, color}) => (
    <span
        style={{
            backgroundColor: color,
            borderRadius: '2px',
            color: '#fff',
            padding: '0.2rem',
        }}>
      {children}
    </span>
);

> Bandwidth allows you to provision specific phone numbers from available inventory for use with the network.

Reviewer: ```suggestion
```

> We offer an API that allows you to search our inventory and purchase numbers for use with our network.

Reviewer: ```suggestion
Bandwidth offers an API that allows you to search our inventory and purchase numbers for use with our network.
```

> In this guide we will show you how to search for available phone numbers and order them through API.

Reviewer: ```suggestion
In this guide we will show you how to search for available phone numbers and order them through the API.
```
